src/main/java/com/synopsys/integration/alert/workflow/scheduled/update/UpdateNotifierTask.java

```diff
@@ -53,14 +53,14 @@ public UpdateNotifierTask(final TaskScheduler taskScheduler, final UpdateChecker
     @Override
     public void runTask() {
         final UpdateModel updateModel = updateChecker.getUpdateModel();
-        if (updateModel.isUpdatable()) {
+        if (updateModel.getUpdatable()) {
             addSystemMessage(updateModel);
             updateEmailService.sendUpdateEmail(updateModel);
         }
     }
 
     private void addSystemMessage(final UpdateModel updateModel) {
-        final String message = String.format("There is a new version of %s available: %s", AlertConstants.ALERT_APPLICATION_NAME, updateModel.getLatestAvailableVersion());
+        final String message = String.format("There is a new version of %s available: %s", AlertConstants.ALERT_APPLICATION_NAME, updateModel.getLatestAvailableTag().getName());
         systemMessageUtility.removeSystemMessagesByType(SystemMessageType.UPDATE_AVAILABLE);
```

Comment:
If we're only using the name here, we might just want to pass the name to the method and not the entire object.